lib/crypto.js

```diff
@@ -247,6 +247,9 @@ ObjectDefineProperty(constants, 'defaultCipherList', {
   value: getOptionValue('--tls-cipher-list')
 });
 
+let webcryptoWarn = true;
+let webcrypto;
```

Comment:
I think we could get by with just one of these variables.

For example:

```js
if (webcrypto === undefined) {
  // warning
}
return webcrypto;
```

or

```js
if (webrcyptoWarn) {
  // warning
}
return lazyRequire(...);
```

Reply:
resolved.